nifi-nar-bundles/nifi-opentsdb-bundle/README.md

```diff
@@ -0,0 +1,59 @@
+# nifi-opentsdb-bundle
```

Comment:
This is great for the GitHub repo but would be even better in an `additionalDetails.html` page so users could access this documentation from their NiFi instance.